contracts/liquidity/Liquidity.sol

```diff
@@ -378,6 +369,61 @@ contract Liquidity is
 		}
 	}
 
+	function _processDirectWithdrawal(
+		WithdrawalLib.Withdrawal memory withdrawal_
+	) internal {
+		TokenInfo memory tokenInfo = getTokenInfo(withdrawal_.tokenIndex);
+
+		bool result = true;
+		if (tokenInfo.tokenType == TokenType.NATIVE) {
+			// solhint-disable-next-line check-send-result
+			bool success = payable(withdrawal_.recipient).send(
+				withdrawal_.amount
+			);
+			result = success;
+		} else if (tokenInfo.tokenType == TokenType.ERC20) {
+			try
+				IERC20(tokenInfo.tokenAddress).transfer(
```

Comment:
Why not use safeTransfer?
I think you could use safeTransfer and wrap it in trycatch.
There are also ERC20s that implement transfer functions that don't return a result.

@kbizikav